portfolio/src/main/webapp/script.js

```diff
@@ -49,3 +49,39 @@ function openLinkedInUrl() {
 function openGitHubUrl() { 
     window.open("https://github.com/RossJ24","_blank");
 }
+
+/**
+ * Fetches previous comments from the server.
+ */
+async function getComments(limit){
+ limit = limit.toString();
+ let res = await fetch('/data?limit='+limit); 
+ let list = await res.json();
+ let text ='';
+ for(obj of list){
+    text += obj+"\n";
+ }
+ addCommentsToDOM(text);
+}
+
+
+/**
+ * Add's the comments to the DOM
+ * @param comments The comments that were fetched from the server 
+ */
+ function addCommentsToDOM(comments){
+    document.getElementById("comment-list").innerText = comments;
+ }
+
+/**
+  *Delete's all comments from the server
+ */
+ async function deleteComments(){
+     console.log("here");
```

Comment:
nit formatting. Also, this log is not descriptive and should indicate a message like 'Inside deleteComments() function'